.gitignore

```diff
@@ -3,4 +3,5 @@ node_modules/
 .vscode
 .env
 
-build/*
\ No newline at end of file
+build/*
+package-lock.json
```

Comment:
Revert this change, ref: https://dev.to/adamklein/package-lock-json-in-git-or-not-50l5